packages/react/src/components/LocalNav/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as LocalNav } from "./LocalNav";
+export { default as LocalNavArgs } from "./LocalNav.args";
```

Comment:
Hey @wc-lei, just reviewing this now but I saw this pattern again in `LocalNav`. Remember that you shouldn't export the args files from component/whatever/index.ts or Rollup will try to bundle it.